### ISSUE TITLE
New repository for Forms

### ISSUE THREAD

**Issue by wsalesky** [biblia-arabica/mss-catalog-pilot#58]
@nathangibson @rvollandt @mMoliere It might be useful to create a new repository for the actual forms application. Where should it live?

**Comment by nathangibson** [comment]
Yes, I think that would make sense. Probably it should either live somewhere in the MAJLIS org or we should create an org for the new Center. What do you think @mMoliere and @rvollandt ?

**Comment by nathangibson** [comment]
@wsalesky We do think that would make sense, but want to discuss #52 with you first.
